FontNode.js

```diff
@@ -0,0 +1,61 @@
+import path from 'path';
+import './lib/inflate.js';
+import './lib/unbrotli.js';
+import './Font.js';
+import fs from 'fs';
+
+function getFontCSSFormat(pth) {
```

Comment:
This looks a lot like the one in `Font.js`, except without the `msg`, is that intended?

Reply:
Good point! I will remove this function and change the one in Font.js accordingly.